stable/redis-ha/values.yaml

```diff
@@ -1,54 +1,86 @@
 ## Configure resource requests and limits
 ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
 ##
-resources:
-  server:
+image: redis
+tag: 4.0.11-stretch
+pullPolicy: IfNotPresent
```

Comment:
This would look better for this set of values:
```yaml
image:
  repository: redis
  tag: 4.0.11-stretch
  pullPolicy: IfNotPresent
```
For reference, that's how image values look when creating a chart with `helm create` and therefore I would recommend doing so in order to keep a common format among the charts.